### PR TITLE
First name game play

### PR DESCRIPTION
**Pull request by henkejosh**
- Added ability to play using first names only

### CODE REVIEW

File: stores/gameState.js [hunk 1]
```diff
@@ -37,6 +40,54 @@ GameState.__onDispatch = function (payload) {
...
+  if (state.status === "incorrect") {
+    state.remedialGuess = true;
+  } else {
+    state.remedialGuess = false;
+  }
+
+  GameState.__emitChange();
+};
+
+var makeFullNameGuess = function (answer) {
```
Comment (wolverdude) [inline stores/gameState.js]: Keep it DRY.

File: stores/gameState.js [hunk 2]
```diff
@@ -26,18 +26,48 @@ var state = {
...
+    case "GUESS_ADDED":
+      makeGuess(payload.guessType, payload.guess);
+      break;
   }
 };
 
+var makeGuess= function(guessType, answer) {
+  var correctAnswer;
+  if (guessType === "Full Name") {
+    correctAnswer = GameState.currentItem().name.toLowerCase();
```
Comment (wolverdude) [inline stores/gameState.js:44]: We should ultimately separate out First & Last name in the data, but for now, this is okay.

**Comment by wolverdude** [comment]
Is this ready?

**Comment by henkejosh** [comment]
Yes

**Pull request merged**